fixtures/webstudio-remix-netlify-edge-functions/app/routes/_index.tsx

```diff
@@ -93,6 +100,13 @@ export const meta: V2_ServerRuntimeMetaFunction<typeof loader> = ({ data }) => {
     });
   }
 
+  if (page.meta.excludePageFromSearch || data?.excludeFromSearch) {
+    metas.push({
+      name: "robots",
+      content: "noindex",
```

Comment:
the other one has also nofollow, forgotten?

Reply:
forgot to regenerate fixtures, cli tests should fail ;)